topics/products/phpstorm/php_open_api.md

```diff
@@ -18,6 +18,36 @@
 
 ## Utility Classes
 
+### `com.jetbrains.php.lang.psi.elements.impl.PhpFilePathUtils`
+
+`PhpFilePathUtils` contains helper methods for working with paths from PSI elements.
+
+#### Useful methods
+
+- `getFileName()` method returns a constant string representation of the path from the PSI element.
```

Comment:
please avoid using terms like "method", "class" etc. in text - it should be clear from notation or context

Reply:
Fixed.